Mailnag/configuration/accountdialog.py

```diff
@@ -172,7 +171,7 @@ def _get_selected_folders(self):
 		folders = []
 		for row in self._liststore_folders:
 			if row[0]:
-				folders.append(mutf7.encode_mutf7(row[1].decode('utf-8')))
+				folders.append(row[1].decode('utf-8'))
```

Comment:
Yes, I have noticed this too. I started to make modifications to accoutdialog.py to support mbox and maildir. I'll try to fix this account member setting also.